extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -90,7 +90,7 @@ export class AmpAdExit extends AMP.BaseElement {
    * @return {function(string): string}
    */
   getUrlVariableRewriter_(args, event, target) {
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.